packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -193,21 +193,15 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
     context: Context,
     did: string
   ): Promise<DIDResolutionResult> {
-    let result
     try {
-      result = await context.did.verifyJWS(commit, {
+      const verification = await context.did.verifyJWS(commit, {
         atTime: meta.timestamp,
+        issuer: did,
         disableTimecheck: meta.disableTimecheck,
       })
+      return verification.didResolutionResult
```

Comment:
I guess there's not really any reason to return anything from this method?

Reply:
Was here before. Decided not to change this.

Reply:
Changed.